docs/resources/aws_s3_bucket_object.md

```diff
@@ -0,0 +1,76 @@
+---
+title: About the aws_s3_bucket_object Resource
+---
+
+# aws_iam_users
+
+Use the `aws_s3_bucket_object` InSpec audit resource to test properties of a s3 bucket object(file).
+
+To test properties of a single object(file), use the `aws_s3_bucket_object` resource.
+
+To test properties of all objects in a bucket, use the `aws_s3_bucket_objects` resource.
+
+<br>
+
+## Syntax
+
+An `aws_s3_bucket_object` resource block users a filter to select a bucket and a object(file) in the bucket
```

Comment:
`filter` is a reserved word in InSpec documentation; it refers to the items that can go in a `.where` clause on a plural resource.  Here, I think you mean resource parameter.